client/source/screens/applications/ApplicationsScreen.tsx

```diff
@@ -0,0 +1,32 @@
+// (c) Tecnologico de Monterrey 2022, rights reserved.
+
+import React from "react";
+import { View, Text } from "react-native";
+import { graphql, useLazyLoadQuery } from "react-relay/hooks";
+import ApplicationList from "../../components/applications/ApplicationList";
+
+const ApplicationsScreen = ({ navigation }): JSX.Element => {
+  const context_id: string = "bcb1f0b7-fbd1-4752-8199-2238e298957a";
+  const data_application_list: any = useLazyLoadQuery(
+    graphql`
+      query ApplicationsScreenQuery($user_id: ID!) {
+        applicationsByUserID(user_id: $user_id) {
+          id
+          title
+        }
+      }
+    `,
+    { user_id: context_id },
+  );
+  const mydata: any = data_application_list;
```

Comment:
Hardcoded IDs **should never be pushed to production**. 

Also, `data_application_list ` and `useLazyLoadQuery` types are missing.